text/0000-ppv.md

```diff
@@ -0,0 +1,1721 @@
+- Feature Name: `portable_packed_vector_types`
+- Start Date: 2020-06-24
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC promotes portable packed SIMD support from the `packed_simd` crate to
+`std::simd` in the standard library.
+
+Future RFCs may expand on the operations and vector sizes provided.
+
+# Acknowledgement
+[acknowledgement]: #acknowledgement
+
+This is forked from [RFC 2366](https://github.com/rust-lang/rfcs/pull/2366) by [gnzlbg](https://github.com/gnzlbg). The main contribution of this fork is the addition of the FAQ section to address question that tend to come up.
+
+# Motivation
+[motivation]: #motivation
+
+The `std::arch` module exposes architecture-specific SIMD types like `_m128` - a
+128-bit wide SIMD vector type. How these bits are interpreted depends on the intrinsic
+being used. For example, let's sum 8 `f32`s values using the SSE4.1 facilities
```

Comment:
This is incorrect to the point of being misleading.

`__m128` (note that it's **two** leading underscores) as well as the higher bit variants, is explicitly and specifically to be interpreted as `f32` lanes.

Similarly, `__m128d` (and wider variants) are specifically `f64` in each lane.

Only the `__m128i` type (and wider variants) holds integer bits where the exact nature of the integer type changes on an operation-by-operation basis.